src/phoenix/trace/langchain/tracer.py

```diff
@@ -351,6 +337,20 @@ def _persist_run(self, run: Run) -> None:
         except Exception:
             logger.exception("Failed to convert run to spans")
 
+    def _null_fallback(
+        self,
+        serialized: Dict[str, Any],
+        messages: List[List[BaseMessage]],
+        *,
+        run_id: UUID,
+        tags: Optional[List[str]] = None,
+        parent_run_id: Optional[UUID] = None,
+        metadata: Optional[Dict[str, Any]] = None,
+        **kwargs: Any,
+    ) -> None:
+        pass
+
+    @graceful_fallback(_null_fallback)
```

Comment:
It doesn't have to be attached to the class of course, but I don't believe the fallback functions should be omitted, they aren't intended to be a guarantee of a suppressed error, but rather a fallback to a less error-prone codepath. In these instances we're just using null fallbacks.

I do consider both functions to kind of go along with one another (the decorated function and its fallback) and is kind of like an implementation detail, not something a class needs to know about. I don't have any kind of preference here

Reply:
Isn't not having a fallback a valid scenario in itself, and isn't that functionally equivalent to having a no-op method? If so, then isn't that (ergonomically) equivalent to just omitting it? I'm not sure if I understand the distinction here.

Reply:
They are the same things yeah, it's just the point of the decorator is to replace a failing function with another that has the same input and return types. While having a `None` return type might be common I don't think it should be the default

Reply:
OK I see, so it's a type issue (and I was barking up the wrong tree). In that case let's move the null method out of the class so people don't get the impression that it's a placeholder instance method (since the input parameters don't matter).